src/QlikConnectorPSExecute/web/connectdialog.ts

```diff
@@ -74,7 +95,7 @@ class ConnectDialog {
             this.connectionInfo = "Please enter a name for the connection.";
         } else {
             if (this.isEdit) {
-                var overrideCredentials = this.username !== "" && this.password !== "";
+                var overrideCredentials = this.username !== this.olduser || this.password.length !== 10;
```

Comment:
sorry, we have to discuss it tomorrow again. In this line the algorithm has to check if the new password is != "**********" and not only the length